openwisp_network_topology/static/netjsongraph/css/src/netjsongraph.css

```
@@ -273,59 +294,62 @@ body {
   margin-top: 20px;
 }
 
-.njg-keyLabel {
+.njg-container .njg-keyLabel {
   font-size: 14px;
   font-weight: 600;
   flex: 1 1 50%;
   word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)